scripts/lib/settings-window-capture/cdp.mjs

```
@@ -0,0 +1,209 @@
+    let message;
+
+    try {
+      message = JSON.parse(typeof event.data === "string" ? event.data : "");
+    } catch {
+      return; // Not something we sent; ignore rather than tear the run down.
+    }
+
+    if (message.id !== undefined && pending.has(message.id)) {
+      const { resolve, reject, timer } = pending.get(message.id);
+      pending.delete(message.id);
+      clearTimeout(timer);
```

ESLint

[error] 'clearTimeout' is not defined.

(no-undef)